target/linux/realtek/image/Makefile

```diff
@@ -67,15 +67,11 @@ define Device/netgear_nge
   IMAGE_SIZE := 14848k
   UIMAGE_MAGIC := 0x4e474520
   DEVICE_VENDOR := Netgear
-+endef
+endef
 
 define Device/netgear_gs110tpp-v1
-  $(Device/Default)
-  SOC := rtl8380
-  IMAGE_SIZE := 14848k
-  UIMAGE_MAGIC := 0x4e474520
-  DEVICE_VENDOR := NETGEAR
-  DEVICE_MODEL := GS110TP
+  $(Device/netgear_nge)
+  DEVICE_MODEL := GS110TPP
```

Comment:
This is not about the device tree. Changes should be moved to the first patch as well.